pkg/operation/botanist/etcd.go

```diff
@@ -108,12 +109,18 @@ func (b *Botanist) DeployEtcd(ctx context.Context) error {
 			return err
 		}
 
+		var backupLeaderElection *config.ETCDBackupLeaderElection
+		if b.Config != nil && b.Config.ETCDConfig != nil {
+			backupLeaderElection = b.Config.ETCDConfig.BackupLeaderElection
```

Comment:
what if `ETCDConfig` don't have `BackupLeaderElection` set ?

Reply:
Then `backupLeaderElection` is still `nil` like before in L112, nothing happens.

Reply:
ok, then at Line no 123 we are setting `LeaderElection` to nil. Will this be an issue?

Reply:
Then `backupLeaderElection` will be `nil`, which should still be safe, or?

Reply:
No, this is no issue and equivalent to not having any configuration for the backup leader election. In this case, `etcd-druid` defaults to whatever its default values are.

Reply:
ok, now I got it. It won’t be an issue.
thanks